onedrive_service/tox.ini

```diff
@@ -21,19 +21,23 @@ envlist =
 [testenv]
 basepython = python3.6
 deps =
+    mock
+    onedrivesdk
     pytest>=3.1,<3.2
 usedevelop = True
-commands = {posargs:py.test -l -vvv '{toxinidir}/tests'}
+commands = {posargs:py.test -l -vvv '{toxinidir}/src/tests'}
 
 [testenv:flake8]
 deps =
     flake8>=3.3,<3.4
     flake8-import-order>=0.12,<0.13
     flake8-quotes>=0.11,<0.12
-commands = {posargs:flake8 '{toxinidir}/setup.py' '{toxinidir}/src' '{toxinidir}/tests'}
+commands = {posargs:flake8 '{toxinidir}/setup.py' '{toxinidir}/src' '{toxinidir}/src/tests'}
 
 [testenv:pylint]
 deps =
+    mock
+    onedrivesdk
```

Comment:
This is not supposed to be here, it is supposed to be in requirements.in/requirements.txt.